src/group/GroupCard.js

```diff
@@ -103,7 +104,7 @@ export default class GroupCard extends PureComponent<Props, State> {
           style={styles.list}
           ownEmail={ownEmail}
           filter=""
```

Comment:
Pass filter to it.
like https://github.com/zulip/zulip-mobile/blob/d1b36684a6f99955699b1a4551d98be1333742db/src/users/UsersCard.js#L31